### PR TITLE
Reader: Add hover transition Comment and Edit buttons

### PR DESCRIPTION
**Pull request by jancavan**
For Reader action buttons, the Edit and Comments button don't have hover transitions while the others do. This PR fixes that.

screenshot 2016-09-23 15 04 55

### CODE REVIEW

File: client/blocks/post-edit-button/index.jsx [hunk 1]
```diff
@@ -13,10 +13,10 @@ import { getEditURL } from 'lib/posts/utils';
 const PostEditButton = ( { post, site, size, onClick, translate } ) => {
 	const editUrl = getEditURL( post, site );
 	return (
-		<a className="post-edit-button" href={ editUrl } onClick={ onClick }>
+		<span className="post-edit-button" href={ editUrl } onClick={ onClick }>
```
Comment (bluefuton) [inline client/blocks/post-edit-button/index.jsx]: Not sure about the change to `span` here.

It's a straightforward link so `a` is the right choice semantically I think.

**Comment by fraying** [comment]
Unless I'm hallucinating, which is a possibility, the hover transitions on fullpost fade in, whereas, on the main stream, they change immediately. They should match. Could we change the fullpost hover to match the one on the main stream?

**Comment by jancavan** [comment]
> the hover transitions on fullpost fade in, whereas, on the main stream, they change immediately.

Hmm, that's odd. I see transitions on both main stream and full-post. Is it a particular button or all of them?

**Comment by fraying** [comment]
Huh. Weird. We're talking about the card footer links on comments and likes, right? On live wordpress.com, I see them change instantly on hover.

Personally, I like the instant change. The fade-in hover makes me feel like my computer is slow.

**Comment by jancavan** [comment]
@fraying Actually, sorry my PR subject is incorrect, just updated it.

Yes, the action links (Edit, Share, Comments, Like, Ellipsis). The transitions have always been there except for Comments (stream) and Comments/Edit (full-post). I think it was just an oversight so I made them all match.

> Personally, I like the instant change.

I don't really have a preference, but since we don't use any transitions on any links, they can be removed for uniformity.

**Comment by bluefuton** [comment]
Looks good to me 🚢

**Comment by jancavan** [comment]
I have removed hover transitions in: https://github.com/Automattic/wp-calypso/pull/8183/commits/42b54ac7c7ddcca4005d6d2da1b0896d12cd28aa

Rebased this branch, but not sure what happened that it misaligned the Share button. Fixed in: https://github.com/Automattic/wp-calypso/pull/8183/commits/b9337cf45b67b00ca3fe4072ad8a9735823781f0

**Pull request merged**